src/outputs.ts

```diff
@@ -0,0 +1,67 @@
+import {DOMWidgetModel, DOMWidgetView} from '@jupyter-widgets/base';
+import {WidgetModel} from '@jupyter-widgets/base';
+import {ViewOptions} from '@jupyter-widgets/base/node_modules/@types/backbone';
+import * as _ from 'underscore';
+
+interface OutputRenderer {
+  renderOutput(modelId: string, container: Element): Promise<void>;
+}
+
+/**
+ * Widget Model for an Output widget.
+ */
+export class OutputModel extends DOMWidgetModel {
+  defaults(): Backbone.ObjectHash {
+    return _.extend(super.defaults(), {
+      outputs: [],
+      _view_name: 'OutputView',
+      _model_name: 'OutputModel',
+      _view_module: '@jupyter-widgets/output',
+      _model_module: '@jupyter-widgets/output',
+    });
+  }
+}
+
+/**
+ * Widget View for an Output widget.
+ */
+export class OutputView extends DOMWidgetView {
+  constructor(options: ViewOptions<WidgetModel>) {
+    super(options);
+
+    this.tagName = 'div';
+  }
+  /**
+   * Called when view is rendered.
+   */
+  /* eslint @typescript-eslint/no-explicit-any: "off" */
+  render(): any {
```

Comment:
this doesn't appear to be returning anything?

Reply:
Updated to return the value of the super call.

The implementation of that is: https://github.com/jupyter-widgets/ipywidgets/blob/be470468f3aa4608c27091acf9b9963f517f513c/packages/base/src/widget.ts#L722